storagenode/operator/operatorserver/static.go

```diff
@@ -0,0 +1,125 @@
+// Code generated by go-bindata. DO NOT EDIT.
+
+package operatorserver
```

Comment:
No embedded data, it will end up bloating our repository. This should be embedded at build stage not here.